### PR TITLE
{bio}[GCCcore/8.3.0] TagDust v2.33

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3578.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e5ae741eac21a0de808edb452916cceb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3104.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/0d5c4e1e5021e0066f3319e23e297e84 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3412.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/ed792a2bc621f2592a92b261a93b3fcd for a full test report.

File: easybuild/easyconfigs/t/tagdust/tagdust-2.33-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# easybuild easyconfig
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+# John Dey <jfdey@fredhutch.org>
+#
+easyblock = 'ConfigureMake'
+
+name = 'tagdust'
```
Comment (boegel) [inline easybuild/easyconfigs/t/tagdust/tagdust-2.33-GCCcore-8.3.0.eb:8]: @fizwit We try to use "official" software names, based on the contents of http://tagdust.sourceforge.net this should be changed to `TagDust`?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
@fizwit I'll take a look once you've addressed @boegel's comment.

**Comment by fizwit** [comment]
@verdurin changed the name to "offical" project name; TagDust

File: easybuild/easyconfigs/t/tagdust/tagdust-2.33-GCCcore-8.3.0.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/t/tagdust/tagdust-2.33-GCCcore-8.3.0.eb]: done in d6c7c8c

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/73f627a990c772399dd1e629255dcc4e for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/229a8cffbfd55d94dbc1f00ca909d347 for a full test report.

**Comment by branfosj** [comment]
> 
> 
> Test report by @boegelbot
> **FAILED**
> Build succeeded for 0 out of 2 (1 easyconfigs in total)
> generoso-c1-s-2 - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz (haswell), Python 3.6.8
> See https://gist.github.com/f129cf24d8f9dc7cf4ee2bc91c58cc7e for a full test report.

Failure caused by running test reports for this and #12773 at the same time and the install lock existing for pandoc.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Going in, thanks @fizwit!

**Comment by branfosj** [comment]
Going in, thanks @fizwit!

**Comment by branfosj** [comment]
Going in, thanks @fizwit!

**Pull request merged**